content/docs/deployment.md

```diff
@@ -166,6 +166,8 @@ ElasticSearch does not require initialization other than
 [installing and running ElasticSearch](https://www.elastic.co/downloads/elasticsearch).
 Once it is running, pass the correct configuration values to the Jaeger collector and query service.
 
+See the [README](https://github.com/jaegertracing/jaeger/tree/master/plugin/storage/es) for an in-depth overview of how Jaeger uses ElasticSearch for storage.
```

Comment:
nit: link to README.md file directly, because otherwise it leads to source code view

Reply:
NB: the readme says "This provides a plugin to use Jaeger with ElasticSearch. This currently supports ElasticSearch 5.x." - we need to mention 6.x too